packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,48 +247,183 @@ function ListView() {
   }
 
   return (
-    <Wrapper>
+    <Layout>
       <PageTitle name="Webhooks" />
-      <Header {...headerProps} />
-      {canRead && (
-        <div className="list-wrapper">
+      <HeaderLayout
```

Comment:
This component should have an `h1` (`<HeaderLayout as="h1" />`) on it and the page itself should have a `<Main labelledBy="the-id-you-put-on-the-h1">`

Hicham managed this the following way in the registration page 😊 


- Concerning the Main component => https://github.com/strapi/strapi/pull/10733/files#diff-5b8fde11bf3bfe8ba7a3b27f0c5e0d3838892702ff8f1d93ab0589f7351d53abR72
- Concerning the title of the page => https://github.com/strapi/strapi/pull/10733/files#diff-5b8fde11bf3bfe8ba7a3b27f0c5e0d3838892702ff8f1d93ab0589f7351d53abR76